### DISCUSSION TITLE
Admin twitter table action column

### DISCUSSION THREAD

**Discussion by ALazyMeme** [pajbot/pajbot#2103] [Polls]
As it stands, the actions column exists whether you're the right level or not (1000+). I believe the table should be collapsing and the action column removed when the user is <level 1000. The downside of this, is a lot of empty space on the right side of the twitter table. However this opens up the page to a streamer-dashboard type thing. Below are some images of how it would look with and without being the appropriate level:

With the level:
![image](https://user-images.githubusercontent.com/12804673/187012029-60f7d370-9801-4e82-8291-4fc30af59465.png)

Without the level:
![image](https://user-images.githubusercontent.com/12804673/187012034-9ebaa094-06bc-49b2-9026-37d881b289ff.png)

**Comment by pajlada** [comment]
I don't think we should remove the column, but we could disable the buttons if we know the user doesn't have permissions to perform the actions
